backend/corpora/common/entities/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .dataset import Dataset  # noqa
```

Comment:
What is this for and why is it noqa?

Reply:
lint will complain that they are imported but not used.

Reply:
Should these be hidden?

I think it's clearer and best practice to specify imports explicitly in each file.

Reply:
I think Entity should be hidden, but I don't think these two should be hidden.